### PR TITLE
DDS-1942 update docs for testing migration

### PR DESCRIPTION
**Pull request by aishling-scilifelab**
## 1. Description / Summary
Updating the user-facing documentation for the migration of the Testing instance. 
Also, in a separate commit, removed a sentence that said there was _no_ risk. Can move this commit to its own branch, if preferred. 

## 2. Jira task / GitHub issue
DDS-1942

## 3. Type of change

What _type of change(s)_ does the PR contain?
- [ ] New feature
  - [ ] Breaking: _Why / How? Add info here._ 
  - [X] Non-breaking 
- [ ] Bug fix 
- [ ] Security Alert fix 
  - [ ] Package update 
    - [ ] Major version update 
- [X] Documentation 
- [ ] Workflow 
- [ ] Tests **only** 

## 4. Additional information

- [X] [Sprintlog](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/SPRINTLOG.md) 
- [ ] Blocking PRs 
  - [ ] Merged 
- [ ] PR to `master` branch: _If checked, read [the release instructions](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/docs/procedures/new_release.md)_ 
  - [ ] I have followed steps 1-8. 

## 5. Actions / Scans

_Check the boxes when the specified checks have passed._

**For information on what the different checks do and how to fix it if they're failing, enter edit mode of this description or go to the [PR template](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/.github/pull_request_template.md).**

- [ ] **Black**

- [ ] **Pylint**

- [ ] **Prettier**

- [ ] **Yamllint**

- [ ] **Tests**

- [ ] **TestPyPI**

- [ ] **CodeQL**

- [ ] **Trivy**

- [ ] **Snyk**

### CODE REVIEW

**Comment by rv0lt** [comment]
I think Sprintlog is always necessary to add an entry to keep track

**Comment by aishling-scilifelab** [comment]
> I think Sprintlog is always necessary to add an entry to keep track

I assume the way to use the sprintlog is: 
- Add a new section with the sprint dates 
- Add a line for each individual PR 

Then, when we do a release we digest the info in the sprint logs into user-facing text for the CHANGELOG.rst?

**Comment by rv0lt** [comment]
Yes, add a new line per PR. With the same format as the other ones. You can run prettier to check the style, otherwise the lint will fail.

Then, on release day. In the changelog will only be mentioned changes that will be seen/affected by the user (ex: Changes on the Actions will not be mentioned there).

**Review by valyo — CHANGES_REQUESTED** [review]
Looks good, only thing needed is a sprintlog entry

**Comment by aishling-scilifelab** [comment]
@valyo @rv0lt SPRINTLOG.md updated.

File: SPRINTLOG.md [hunk 1]
```diff
@@ -349,3 +349,7 @@ _Empty sprint_
 # 2024-04-8 - 2024-04-19
 
 - New version: 2.6.4 ([#687](https://github.com/ScilifelabDataCentre/dds_cli/pull/687))
+
+# 2024-05-20 - 2024-06-03
```
Comment (valyo) [inline SPRINTLOG.md:353]:
```suggestion
# 2024-05-20 - 2024-05-31
```
for consistency's sake (we've always had the sprints from starting monday till ending friday)

**Review by valyo — CHANGES_REQUESTED** [review]

File: SPRINTLOG.md (hunk 1 above)
Comment (aishling-scilifelab) [reply SPRINTLOG.md]: Ah copied the dates from the Jira sprint dates. Fixed now 👍

**Review by valyo — APPROVED** [review]

**Review by rv0lt — APPROVED** [review]

**Pull request merged**